packages/lesswrong/components/sequences/ChaptersEditForm.tsx

```diff
@@ -29,15 +35,26 @@ const ChaptersEditForm = ({classes, documentId, postIds, successCallback, cancel
   cancelCallback: any,
 }) => {
   const { openDialog } = useDialog();
+  const [saved, setSaved] = useState(true);
+  const { flash } = useMessages();
+
+  const changeCallback = useCallback((doc: ChaptersFragment) => {
+    setSaved(_.isEqual(doc.postIds, postIds));
```

Comment:
I think we prefer `lodash` where possible over `underscore` (& import only the necessary module, i.e. `import isEqual from 'lodash/isEqual';`), but minor nit